tsconfig.json

```diff
@@ -3,19 +3,19 @@
     "allowSyntheticDefaultImports": false,
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
-    "alwaysStrict": true,
+//    "alwaysStrict": true,
```

Comment:
Yes that is correct.  There are nine PRs in total with the PRs at the end re-enabling the compiler options.